packages/codemods/src/transforms/styled-components-to-compiled/__tests__/styled-components-to-compiled.test.tsx

```diff
@@ -178,7 +178,7 @@ describe('styled-components-to-compiled transformer', () => {
         {
           create: (_: FileInfo) => ({
             transform: {
-              buildImport: ({ originalNode, currentNode }) => {
+              buildImport: ({ originalNode, currentNode }: any) => {
```

Comment:
Should just be `BuildImportContext`

Reply:
Thanks! What would the type be to pass in?

Reply:
I'm not actually sure, I have wanted to remove the `T` since it doesn't seem necessary to me anymore. Can you help out here @JakeLane since you initially wrote that `ValidateConfig` type?